### PR TITLE
[stable/redis] Expose redis metrics service port

### PR DESCRIPTION
**Pull request by MichaelPalmer1**
When metrics are enabled, the port should be exposed on the service. Otherwise, it is impossible to create a proxy via the API to the metrics service (i.e. if your Prometheus instance is not in the same Kubernetes environment).

## Release Notes
```release-note
- Exposed the redis metrics port
```

### CODE REVIEW

**Comment by MichaelPalmer1** [comment]
CLA has been signed.

File: stable/redis/templates/svc.yaml [hunk 1]
```diff
@@ -24,5 +24,10 @@ spec:
   - name: redis
     port: 6379
     targetPort: redis
+  {{- if .Values.metrics.enabled }}
+  - name: metrics
+    port: 9121
+    targetPort: redis
```
Comment (unguiculus) [inline stable/redis/templates/svc.yaml]: Is the target port really `redis`, i. e. 6379?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/redis/templates/svc.yaml (hunk 1 above)
Comment (MichaelPalmer1) [reply stable/redis/templates/svc.yaml]: You're right. I've updated it.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**